tests/test_monitor.py

```diff
@@ -85,9 +85,23 @@ def test_remove_monitor(hlwm):
     assert hlwm.get_attr('monitors.focus.name') == 'monitor2'
 
 
+def test_remove_monitor_invalid_args(hlwm):
+    hlwm.call_xfail('remove_monitor') \
+        .expect_stderr('Expected one argument, but got only 0')
+
+    hlwm.call('add tag2')
+    hlwm.call('add_monitor 800x600+40+40 tag2 monitor2')
+
+    hlwm.call_xfail('remove_monitor foobar') \
+        .expect_stderr('No such monitor: foobar')
+
+    hlwm.call_xfail('remove_monitor 2') \
+        .expect_stderr('No such monitor: 2')
+
+
 def test_cannot_remove_nonexistent_monitor(hlwm):
```

Comment:
Isn't this test a bit redundant now?

Reply:
Right,.. I've removed it and added asserts in the `test_remove_monitor_invalid_args`

Reply:
Thanks!